extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

```diff
@@ -0,0 +1,95 @@
+define('wikia.articleVideo.featuredVideo.ads', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.video.vastUrlBuilder',
+	'ext.wikia.adEngine.slot.service.megaAdUnitBuilder',
+	'wikia.log'
+], function (adContext, vastUrlBuilder, megaAdUnitBuilder, log) {
+
+	var aspectRatio = 640 / 480,
+		logGroup = 'wikia.articleVideo.featuredVideo.ads';
+
+	function calculateRV(depth) {
+		var capping = adContext.get('opts.fvAdsFrequency');
+
+		return depth < 2 || !capping ? 1 : Math.floor((depth - 1) / capping) + 1;
+	}
+
+	function cappingAllowsToShowNextVideo(depth) {
```

Comment:
it isn't only capping that doesn't allow to play video. Maybe `configAllowsToShowNextVideo()` ?